### PR TITLE
Create Twig\AssetExtension and image Twig function

### PR DESCRIPTION
**Pull request by MauricioFauth**
Removes the need to pass `$PMA_Theme->getImgPath()` to templates.

### CODE REVIEW

File: libraries/classes/Twig/AssetExtension.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+final class AssetExtension extends AbstractExtension
+{
+    /**
+     * @return TwigFunction[]
+     */
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('image', [$this, 'getImagePath']),
```
Comment (williamdes) [inline libraries/classes/Twig/AssetExtension.php:19]: theme_image could be easier to maintain and search for?

**Review by williamdes — APPROVED** [review]
Very good work

File: libraries/classes/Twig/AssetExtension.php (hunk 1 above)
Comment (mauriciofauth) [reply libraries/classes/Twig/AssetExtension.php:19]: As all images are from the themes, I thought that `image` is simpler. I thought about using `image_asset` as well.

Comment (williamdes) [reply libraries/classes/Twig/AssetExtension.php:19]: I am convinced, you can merge when you want :)

**Pull request merged**